defs/bitcoin/qtum.json

```diff
@@ -0,0 +1,41 @@
+{
+    "coin_name": "Qtum",
+    "coin_shortcut": "QTUM",
+    "coin_label": "Qtum",
+    "website": "https://qtum.org",
+    "github": "https://github.com/qtumproject/qtum",
+    "maintainer": "CodeFace <codeface@qtum.info>",
+    "curve_name": "secp256k1",
+    "address_type": 58,
+    "address_type_p2sh": 50,
+    "maxfee_kb": 40000000,
+    "minfee_kb": 400000,
+    "signed_message_header": "Qtum Signed Message:\n",
+    "hash_genesis_block": "000075aef83cf2853580f8ae8ce6f8c3096cfa21d98334d6e3f95e5582ed986c",
+    "xprv_magic": 76066276,
+    "xpub_magic": 76067358,
+    "xpub_magic_segwit_p2sh": 77429938,
+    "xpub_magic_segwit_native": 78792518,
+    "bech32_prefix": "qc",
+    "cashaddr_prefix": null,
+    "slip44": 88,
```

Comment:
per [SLIP-44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md) this value should be 2301

Reply:
Hi @matejcik, the 2301 is an incorrect number, see the discussion here https://github.com/satoshilabs/slips/pull/196

Reply:
could be, but per the same discussion and https://github.com/satoshilabs/slips/pull/120 number 88 is already taken and we don't allow duplicates in our codebases.

Reply:
changed to 2301